index.js

```diff
@@ -38,6 +38,10 @@ var validate = module.exports = function (name) {
     errors.push('name cannot start with an underscore')
   }
 
+  if (name.length && !name.match(/^[a-z0-9]/)) {
+    warnings.push('name cannot start with a special/capital character')
```

Comment:
instead of `capital character`, either use `uppercase character` or `uppercase letter` or `capital letter` t/o

```suggestion
    warnings.push('name cannot start with a special or uppercase character')
```